app/jobs/regular/discourse_automation_trigger.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DiscourseAutomationTrigger < ::Jobs::Base
+    def execute(args)
+      automation = DiscourseAutomation::Automation.find_by(id: args[:automation_id])
```

Comment:
you probably want to find by `enabled: true` too, might want to make it a default scope, but I have been bitten too many times in my life by default scopes...